scripts/setup-shaper-calibrate.sh

```diff
@@ -0,0 +1,10 @@
+# This script instals the prerequisites for measureing resonances on a standard, single instance install of Klippy
+
+# Step 1: Install numpy in the virtualenv so it can be used by Klippy
```

Comment:
It might be great to actually echo these messages to the command line. Otherwise, almost no-one would read these comments.

Reply:
Good idea